python/ray/tune/examples/dcgan_mnist_pbt/dcgan_mnist_pbt.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+from ray import tune
+from ray.tune.schedulers import PopulationBasedTraining
+
+import argparse
+import os
+import random
+import torch
+import torch.nn as nn
+import torch.nn.parallel
+import torch.optim as optim
+import torch.utils.data
+import torchvision.datasets as dset
+import torchvision.transforms as transforms
+import torchvision.utils as vutils
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+
+from torch.autograd import Variable
+from torch.nn import functional as F
+import torch.utils.data
+from scipy.stats import entropy
+
+# Set random seed for reproducibility
+manualSeed = 999
+# manualSeed = random.randint(1, 10000) # use if you want new results
+print("Random Seed: ", manualSeed)
+random.seed(manualSeed)
+torch.manual_seed(manualSeed)
```

Comment:
btw, does this actually work? I think the seed needs to be set per trainable (i.e., in _setup)

Reply:
remove it for now.